tests/foreman/destructive/test_contentview.py

```diff
@@ -81,14 +80,14 @@ def test_positive_reboot_recover_cv_publish(target_sat, function_entitlement_man
     try:
         publish_task = cv.publish(synchronous=False)
         target_sat.power_control(state='reboot', ensure=True)
-        wait_for_tasks(
+        target_sat.api_factory.wait_for_tasks(
             search_query=(f'id = {publish_task["id"]}'),
             search_rate=30,
             max_tries=60,
         )
     except TaskFailedError:
         target_sat.api.ForemanTask().bulk_resume(data={"task_ids": [publish_task['id']]})
-        wait_for_tasks(
+        target_sat.api_factory.wait_for_tasks(
```

Comment:
Wait for task is part of Capsule mixins, not api_factory, right?
```suggestion
        target_sat.wait_for_tasks(
```